pyomo/core/base/PyomoModel.py

```diff
@@ -829,6 +825,8 @@ def _load_model_data(self, modeldata, namespaces, **kwds):
 
                 self._initialize_component(modeldata, namespaces, component_name, profile_memory)
                 if False:
+                    # !!THIS SEEMS LIKE A BUG!! - mrmundt
+                    # there is no start_time
```

Comment:
Agreed, this whole if-block is unreachable and likely there for reference. @jsiirola looks like you were the last one to touch these lines (3 years ago), can this if-block be removed?

Reply:
Yes.  I think thus can be removed.  My guess is that this was the old "construction time reporting" that was superseded by the new timing reporting in `pyomo.common.timing`.